### DISCUSSION TITLE
Customize the name for note events

### DISCUSSION THREAD

**Discussion by LilaRest** [davish/obsidian-full-calendar#378] [Feature Requests]
Hi !
Actually when using "Full note" calendar, events' notes are named from the following format "YYYY-MM-DD {{title}}".
For consistency reasons I would have loved to name those ones "E-{{number}}" where `number` is the number of the event. Also I imagine that not everyone will love that default format, eg. some people may want to localize the date format.

So it could be great to being able to define a custom format for newly created events :)

Thanks in advance,
Lilian.

**Comment by LilaRest** [comment]
Hi @davish ! Thanks for sorting this issue. Have you planned to work on it ?

**Comment by davish** [comment]
Hi @0Lilian, I'm currently working on a pretty significant rewrite of the core plugin logic in #204. After that I will likely be addressing feature requests by popularity, as determined by the number of reactions on the issue.

**Comment by LilaRest** [comment]
> Hi @0Lilian, I'm currently working on a pretty significant rewrite of the core plugin logic in #204. After that I will likely be addressing feature requests by popularity, as determined by the number of reactions on the issue.

Ok, thanks for your answer and thanks for that great plugin :)
